packages/stark-ui/src/modules/slider/components/slider.component.ts

```diff
@@ -54,21 +54,21 @@ export class StarkSliderComponent extends AbstractStarkUiComponent implements Af
 	 * Configuration object for the slider instance to be created.
 	 */
 	@Input()
-	public sliderConfig: StarkSliderConfig;
+	public sliderConfig: StarkSliderConfig = { range: { max: [], min: [] } };
```

Comment:
I'm not sure if the slider will work correctly if we define this default range...